### DISCUSSION TITLE
Warning: Source is directory

### DISCUSSION THREAD

**Discussion by Nef10** [BetaHuhn/repo-file-sync-action#90] [Q&A] [answered]
Hey,

I just wanted to ask why I get a warning `Source is directory` - from the readme it looks like this is a supported workflow? Is there anything special to take into consideration? Otherwise should this rather be a notice?

**Comment by BetaHuhn** [comment, marked as answer]
This is a supported workflow and works out of the box. 

You are right, it should be a simple notice instead of a warning. Will be fixed in the next version.
